doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md

```diff
@@ -81,14 +96,14 @@ PostgreSQL [tables name length](http://www.postgresql.org/docs/current/static/sq
 
 The following table summarizes the table name composition (old encoding):
 
-| FIWARE service path | `dm-by-service-path` | `dm-by-entity` | `dm-by-entity-type` |
+| FIWARE service path | `dm-by-service-path` | `dm-by-entity`, `data_model=dm-by-entity-database-schema`, `data_model=dm-by-entity-database` | `dm-by-entity-type` |
 |---|---|---|---|
 | `/` | N/A | `<entityId>_<entityType>` | `<entityType>` |
 | `/<svcPath>` | `<svcPath>` | `<svcPath>_<entityId>_<entityType>` | `<svcPath>_<entityType>` |
 
 Using the new encoding:
 
-| FIWARE service path | `dm-by-service-path` | `dm-by-entity` | `dm-by-entity-type` |
+| FIWARE service path | `dm-by-service-path` | `dm-by-entity`, `data_model=dm-by-entity-database-schema`, `data_model=dm-by-entity-database` | `dm-by-entity-type` |
```

Comment:
The `data_model=` maybe should be removed?

Reply:
Fixed in [9bcedc4](https://github.com/telefonicaid/fiware-cygnus/pull/1876/commits/9bcedc4805e6b3e4e48b7e5a357e18931cd62a0a)